ale_linters/python/mypy.vim

```diff
@@ -7,6 +7,29 @@ call ale#Set('python_mypy_show_notes', 1)
 call ale#Set('python_mypy_options', '')
 call ale#Set('python_mypy_use_global', get(g:, 'ale_use_global_executables', 0))
 call ale#Set('python_mypy_auto_pipenv', 0)
+call ale#Set('python_mypy_run_even_with_dmypy', 0)
+
+function! ale_linters#python#mypy#upgadeToDmypyIfPossible(buffer) abort
```

Comment:
To make this work, you can return `0` or `1` from this function, and call it in `ale_linters#python#mypy#GetExecutable`. You can return an empty string from that function to skip running the linter.